### PR TITLE
Switch to SVGs for most icons

### PR DESCRIPTION
**Pull request by timja**
# Description

A brief summary describing the changes in this pull request. See 
[JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX) for further information. 

<details>
<summary>Light branch</summary>

image

</details>

<details>
<summary>Dark branch</summary>

image

</details>

<details>
<summary>Light repo</summary>

image

</details>

<details>
<summary>Dark repo</summary>

image

</details>

<details>
<summary>Light github logo</summary>

image

</details>

<details>
<summary>Dark github logo</summary>

image

</details>

Fixes https://github.com/jenkinsci/dark-theme-plugin/issues/104

Requires https://github.com/jenkinsci/jenkins/pull/5065

I've left the scm navigator icon alone, it's only used on the new job page as far as I could tell, I didn't think any of the icons I found were a good fit

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Automated tests have been added to exercise the changes
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verify that the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

# Documentation changes
- [ ] Link to jenkins.io PR, or an explanation for why no doc changes are needed

# Users/aliases to notify

@bitwiseman @dwnusbaum @fqueiruga

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
         <changelist>-SNAPSHOT</changelist>
         <hpi.compatibleSinceVersion>2.2.0</hpi.compatibleSinceVersion>
         <java.level>8</java.level>
-        <jenkins.version>2.271</jenkins.version>
+        <jenkins.version>2.281-SNAPSHOT</jenkins.version> <!-- TODO https://github.com/jenkinsci/jenkins/pull/5065 -->
```
Comment (timja) [inline pom.xml:28]: TODO: use incremental once build completes

File: src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java [hunk 2]
```diff
@@ -1585,54 +1586,54 @@ public boolean test(SCMTraitDescriptor<?> scmTraitDescriptor) {
 
             IconSet.icons.addIcon(
                     new Icon("icon-github-logo icon-sm",
-                            "plugin/github-branch-source/images/16x16/github-logo.png",
-                            Icon.ICON_SMALL_STYLE));
+                            "plugin/github-branch-source/images/svgs/sprite-github.svg#github-logo",
```
Comment (timja) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java:1589]: I've left the old github-logo image file in as it's used here: https://github.com/jenkinsci/github-branch-source-plugin/blob/42a2234b2a501f299f10206bad81c2ff9a87cd8d/src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubOrgMetadataAction.java#L82

I'm not aware of a way to trigger that code though

File: pom.xml (hunk 1 above)
Comment (timja) [reply pom.xml:28]:
```suggestion
        <jenkins.version>2.281-rc30929.f24d46502521</jenkins.version> <!-- TODO https://github.com/jenkinsci/jenkins/pull/5065 -->
```

File: src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java (hunk 2 above)
Comment (bitwiseman) [reply src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java:1589]: Could we return the same SVG regardless of `size` value?

**Comment by bitwiseman** [comment]
This looks good to me.

File: src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java (hunk 2 above)
Comment (timja) [reply src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java:1589]: Yes we can, I'm just wary of touching code I can't test.

My point was I don't know if this code is possible to hit, at least on github.com

File: pom.xml [hunk 3]
```diff
@@ -25,7 +25,7 @@
         <changelist>-SNAPSHOT</changelist>
         <hpi.compatibleSinceVersion>2.2.0</hpi.compatibleSinceVersion>
         <java.level>8</java.level>
-        <jenkins.version>2.271</jenkins.version>
+        <jenkins.version>2.281-rc30929.f24d46502521</jenkins.version> <!-- TODO https://github.com/jenkinsci/jenkins/pull/5065 -->
```
Comment (timja) [inline pom.xml:28]:
```suggestion
        <jenkins.version>2.283</jenkins.version>
```

File: pom.xml [hunk 4]
```diff
@@ -117,8 +117,8 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.263.x</artifactId>
-                <version>23</version>
+                <artifactId>bom-2.277.x</artifactId>
+                <version>24</version>
```
Comment (timja) [inline pom.xml:121]:
```suggestion
                <version>27</version>
```

**Comment by bitwiseman** [comment]
@timja What are we waiting on here?

**Comment by timja** [comment]
nothing

**Review by bitwiseman — APPROVED** [review]

**Comment by timja** [comment]
@bitwiseman can this get re-released please?

**Comment by bitwiseman** [comment]
@timja Updating to use the 2.289.x LTS.  This means we now have three release lines. :sob:

**Pull request merged**